### PR TITLE
ssh-copy-id 8.2p1

### PR DESCRIPTION
**Pull request by EricFromCanada**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/ssh-copy-id.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class SshCopyId < Formula
   desc "Add a public key to a remote machine's authorized_keys file"
   homepage "https://www.openssh.com/"
-  url "https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-8.1p1.tar.gz"
-  mirror "https://mirror.vdms.io/pub/OpenBSD/OpenSSH/portable/openssh-8.1p1.tar.gz"
-  version "8.1p1"
-  sha256 "02f5dbef3835d0753556f973cd57b4c19b6b1f6cd24c03445e23ac77ca1b93ff"
+  url "https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-8.2p1.tar.gz"
+  version "8.2p1"
```
Comment (SMillerDev) [inline Formula/ssh-copy-id.rb]: You removed the mirror here.

**Review by SMillerDev — COMMENTED** [review]
Please put the mirror back

**Comment by EricFromCanada** [comment]
It'd be good for `bump-formula-pr` to alert when a mirror URL is omitted, since it's usually accidental.

**Comment by SMillerDev** [comment]
Agreed, maybe if we ask @homebrew/brew nicely?

**Pull request closed without merging**